test/Manager.t.sol

```diff
@@ -53,21 +53,20 @@ contract EndpointAndManagerContract is EndpointAndManager, IEndpointReceiver {
         uint64 rateLimitDuration
     ) EndpointAndManager(token, mode, chainId, rateLimitDuration) {}
 
-    function _quoteDeliveryPrice(uint16 /* recipientChain */ )
-        internal
-        pure
-        override
-        returns (uint256)
-    {
+    function _quoteDeliveryPrice(
+        address, /* token */
+        uint16 /* recipientChain */
+    ) internal pure override returns (uint256) {
```

Comment:
can you make these comments docstrings pls?

Reply:
Discussed offline -- this is just personal preference but I like having the variables commented out on the side for tests, so if someone wants to start using them they easily can just by commenting out and the rest are clearly there. I don't think we really should need to have the `@param` docstrings for the test methods.

But I do agree we should have the @param docstrings generally in the base level interfaces and abstract contracts. That's something to tackle as part of the docstrings issue #24